daprdocs/content/en/concepts/dapr-services.md

```diff
@@ -0,0 +1,45 @@
+---
+type: docs
+title: "Dapr Services"
+linkTitle: "Services"
+weight: 100
+---
+
+
+Dapr supports multiple services that help sidecar - sidecar, sidecar - app and sidecar - system communication. Following are the services and the way to leverage them with your application.
+
+#### Daprd
+
+Dapr sidecar is available as Kubernetes container or as a Daprd process while using Dapr locally. Dapr runs Daprd by default when `Dapr Run` is executed through Dapr cli. While that's the default behavior, some users might still have to use Daprd explicity in certain special scenarios. For example a user may want to run sidecar in production system without running in a container. In cases like that when daprd can be executed through command line as follows
+
+```bash
+~/.dapr/bin/daprd -app-id <app-name>
```

Comment:
It would be good to add multiple examples of a common use showing how arguments are used and explaining the scenario (e.g. example for running a Dapr sidecar with gRPC port set to X, mTLS enabled and log-level Y with another example showing something different.)